pkgs/development/libraries/libcamera/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, fetchgit
+, lib
+, meson
+, ninja
+, pkg-config
+, boost
+, gnutls
+, openssl
+, libevent
+, lttng-ust
+, libtiff
+, qt5
+, gst_all_1
+, graphviz
+, doxygen
+, python3
+, python3Packages
+}:
+
+stdenv.mkDerivation {
+  pname = "libcamera";
+  version = "1.0.0";
+
+  src = fetchgit {
+    url = "git://linuxtv.org/libcamera.git";
+    rev = "040a6dcfbccaf1005da2c571769b1f89df1f07cf";
+    sha256 = "sha256-RCNKFCv3fNabroCUR5MtKbyDS20J7leU8KkE1U5qe1M=";
+  };
+
+  postPatch = ''
+    patchShebangs utils/
+  '';
+
+  buildInputs = [
+    # core requirements
+    python3Packages.pyyaml
```

Comment:
These seem to be build-time requirements AFAICT so I would maybe try to move them inside `python3.withPackages` in `nativeBuildInputs`.

Reply:
For some reason `python3.withPackages` does not work at all here - I get `ModuleNotFoundError: No module named 'jinja2'`, but I can confirm if I at least move the packages into `nativeBuildInputs` the build still works. I've done this.